Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -453,18 +483,6 @@ void GcodeSuite::G33() {
 
   SERIAL_ECHOLNPGM("G33 Auto Calibrate");
 
-  const float dcr = delta_calibration_radius();
-
-  if (!_1p_calibration && !_0p_calibration) { // test if the outer radius is reachable
```

Comment:
Since the calculation of the default value for delta calibration radius and changing this value by the user are now done within G33 safety tests are now done during parsing of the G33 command.